src/Lachain.Console/TrustedKeygen.cs

```diff
@@ -135,7 +135,7 @@ internal class Config
                     Hosts = new[] {"+"},
                     Port = basePort,
                     MetricsPort = (ushort) (basePort + 1),
-                    ApiKey = "asdasdasd",
+                    ApiKey = "0x2e917846fe7487a4ea3a765473a3fc9b2d9227a4d312bc77fb9de357cf73d7e52b771d537394336e9eb2cb4838138f668f4bd7d8cf7e04d9242a42c71b99f166",
```

Comment:
We meed to generate random key here

Reply:
And print associated private key to logs to have ability to save it for future use in client

Reply:
Fixed in commit #170 8421408a